Gifski/Intents.swift

```diff
@@ -86,29 +316,44 @@ struct ConvertIntent: AppIntent, ProgressReportingIntent {
 	)
 	var dimensionsHeight: Int?
 
+	@Parameter(
+		title: "Crop",
+		description: "Optionally crop the video.",
+	)
+	var crop: CropEntity?
+
+	@Parameter(
+		title: "Output Mode",
+		description: "Output an animated GIF or a single frame preview of the GIF.",
+		default: .animatedGIF
+	)
+	var outputMode: ConvertOutputMode
+
 	// TODO: Dimensions setting. Percentage or width/height.
 
 	static var parameterSummary: some ParameterSummary {
 		Switch(\.$dimensionsType) {
 			Case(.pixels) {
-				Summary("Convert \(\.$video) to animated GIF") {
+				Summary("Convert \(\.$video) to \(\.$outputMode)") {
```

Comment:
It's not important enough to put in the summary. It can just be the last parameter after `$crop`.